src/ProxyKit/HttpContextExtensions.cs

```diff
@@ -53,7 +53,8 @@ private static HttpRequestMessage CreateProxyHttpRequest(this HttpRequest reques
             var requestMessage = new HttpRequestMessage();
             
             //Only copy Body when original request has a body.
-            if (request.ContentLength.HasValue)
+            if (//request.Method != "GET" || 
```

Comment:
This solves it for me.. I believe this should be done only for gets as this was the original problem?